test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -707,6 +707,21 @@ INI conv_multi_stride(
   }
 );
 
+INI conv_uneven_strides(
+  "conv_uneven_strides",
+  {
+    nn_base + "learning_rate=0.1 | optimizer=sgd | loss=cross | batch_size=3",
```

Comment:
FYI: if #1032 is merged before this PR, this unittest will need an update.

Reply:
Acked.